org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/ZkReportViewer.java

```diff
@@ -2066,4 +2069,11 @@ private void setupFindwindow(FindWindow findWindow) {
 		findWindow.setSizable(false);
 		findWindow.setContentStyle("background-color: #fff; width: 99%; margin: auto;");
 	}
+	
+	/**
+	 * Update Row Count label
+	 */
+	private void updateRowCount() {
+		rowCount.setValue(Msg.getMsg(Env.getCtx(), "RowCount", new Object[] {m_reportEngine.getPrintData().getRowCount(false)}));

```

Comment:
probably safer to add a null check for rowCount here.

Reply:
ok, changes applied